testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -348,20 +354,21 @@ def test_AlignTraj_in_memory(self, universe, reference, tmpdir):
         x = align.AlignTraj(universe, reference, filename=outfile,
                             in_memory=True).run()
         assert x.filename is None
-        assert_almost_equal(x.results.rmsd[0], 6.9290, decimal=3)
-        assert_almost_equal(x.results.rmsd[-1], 5.2797e-07, decimal=3)
+        assert_allclose(x.results.rmsd[0], 6.9290, rtol=0, atol=1.5*(1e-3))
+        assert_allclose(x.results.rmsd[-1], 5.2797e-07, rtol=0, atol=1.5*(1e-3))
 
         # check in memory trajectory
         self._assert_rmsd(reference, universe, 0, 6.929083044751061)
         self._assert_rmsd(reference, universe, -1, 0.0)
 
     def _assert_rmsd(self, reference, fitted, frame, desired, weights=None):
-        fitted.trajectory[frame]
-        rmsd = rms.rmsd(reference.atoms.positions, fitted.atoms.positions,
-                        superposition=True)
-        assert_almost_equal(rmsd, desired, decimal=5,
-                            err_msg="frame {0:d} of fit does not have "
-                                    "expected RMSD".format(frame))
+            fitted.trajectory[frame]
```

Comment:
You still have unwanted whitespace changes here and elsewhere, please re-read the entire diff line-by-line and make sure those changes are not present (since I'm spending time doing the same).